roles/openshift_storage_glusterfs/tasks/main.yml

```diff
@@ -0,0 +1,192 @@
+---
+- name: Create temp directory for doing work in
+  command: mktemp -d /tmp/openshift-glusterfs-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+  check_mode: no
+
+- name: Copy the admin client config(s)
```

Comment:
If you are using kubeconfig via the ``oc_*``  modules you no longer need to do this.